pkg/k8shandler/defaults.go

```diff
@@ -0,0 +1,34 @@
+package k8shandler
+
+import (
+	"fmt"
+)
+
+const (
+	modeUnique    = "unique"
+	modeOpsShared = "ops_shared"
+	defaultMode   = modeUnique
+)
+
+func KibanaIndexMode(mode string) (string, error) {
+	if mode == "" {
+		return defaultMode, nil
+	}
+	if mode == modeUnique || mode == modeOpsShared {
+		return mode, nil
+	}
+	return "", fmt.Errorf("Invalid kibana index mode provided [%s]", mode)
+}
+
+// 'es' or 'es-ops'
+func EsUnicastHost(opsCluster string) string {
+	return fmt.Sprintf("logging-%s-cluster", opsCluster)
+}
+
+func AllowClusterReader() bool {
+	return false
+}
+
+func RootLogger() string {
+	return "console"
```

Comment:
lets make this `rolling` instead as our default

Reply:
Do you mean `file`?